### PR TITLE
[Azure Analysis Services] Fix swagger coverage by adding three operations

### PR DESCRIPTION
**Pull request by taiwu**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by dsgouda** [comment]
Please remove the npm-debug.log file and fix semantic errors flagged [here](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/302056376). Looks like some of the changes you made are invalid.
I will review the changes once this is fixed.

**Comment by dsgouda** [comment]
@taiwu there are some pending semantic validation errors, can you take a look?

File: specification/analysisservices/resource-manager/Microsoft.AnalysisServices/2016-05-16/analysisservices.json [hunk 1]
```diff
@@ -587,6 +713,82 @@
...
+        "endTime": {
+          "type": "string",
+          "description": "The end time of the operation."
+        },
+        "status": {
+          "type": "string",
+          "description": "The status of the operation."
+        },
+        "error": {
+          "type": "object",
```
Comment (dsgouda) [inline specification/analysisservices/resource-manager/Microsoft.AnalysisServices/2016-05-16/analysisservices.json]: Move this object definition to its own block

File: specification/analysisservices/resource-manager/Microsoft.AnalysisServices/2017-08-01-beta/analysisservices.json [hunk 2]
```diff
@@ -481,6 +481,141 @@
...
+            "type": "string",
+            "description": "The region name which the operation will lookup into."
+          },
+          {
+            "name": "serverParameters",
+            "in": "body",
+            "required": true,
+            "schema":
+            { 
+              "$ref": "#/definitions/CheckServerNameAvailabilityBody"
```
Comment (dsgouda) [inline specification/analysisservices/resource-manager/Microsoft.AnalysisServices/2017-08-01-beta/analysisservices.json]: A better name is `CheckServerNameAvailabilityParameters`

**Review by dsgouda — COMMENTED** [review]
Apart from a couple of minor comments, looks okay

File: specification/analysisservices/resource-manager/Microsoft.AnalysisServices/2017-08-01-beta/analysisservices.json (hunk 2 above)
Comment (taiwu) [reply specification/analysisservices/resource-manager/Microsoft.AnalysisServices/2017-08-01-beta/analysisservices.json]: Thanks for the review, I updated them, please take a look once the build is completed.

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**